iphone/Classes/TiLogServer.m

```diff
@@ -223,6 +242,23 @@ +(void)startServer
 		connections = [[NSMutableArray alloc] init];
 	}
 
+	if (headers == nil) {
+		NSString* version = [NSString stringWithCString:TI_VERSION_STR encoding:NSUTF8StringEncoding];
+		NSDictionary* map = [[NSDictionary alloc] initWithObjectsAndKeys:
```

Comment:
Oh, so `[NSDictionary dictionaryWithObjectsAndKeys ...]` is essentially a static method. That's cool. Want me to update it?